huaweicloud/resource_huaweicloud_compute_eip_associate.go

```diff
@@ -95,7 +95,7 @@ func resourceComputeEIPAssociateCreate(d *schema.ResourceData, meta interface{})
 	// get port id
 	portID, privateIP, err := getComputeInstancePortIDbyFixedIP(d, config, instanceID, fixedIP)
 	if err != nil {
-		return fmtp.Errorf("Error getting port id of compute instance: %s", err)
+		return common.CheckDeleted(d, err, "eip associate")
```

Comment:
seems we should just raise error within  Create

Reply:
done.